src/Devices/PushingBox.cpp

```diff
@@ -69,5 +63,12 @@ void PushingBox::sendData() {
   float temperature = TempProbe_TC::instance()->getRunningAverage();
   float pH = PHProbe::instance()->getPh();
   snprintf(buffer, sizeof(buffer), format, DevID, tankID, temperature, pH);
-  client.write(buffer, strnlen(buffer, sizeof(buffer)));
+  serial(buffer);
+  serial("attempting to connect to PushingBox...");
+  if (client.connected() || client.connect(server, 80)) {
```

Comment:
Why move this code to later in the function?

Reply:
I wanted to see the data that was generated to send to PushingBox even if we could not connect. The problem being investigated was that we often have zero and this allowed me to see what would have been sent.